L1Trigger/L1THGCal/plugins/veryfrontend/HGCalVFEProcessorSums.cc

```diff
@@ -0,0 +1,100 @@
+#include "L1Trigger/L1THGCal/interface/veryfrontend/HGCalVFEProcessorSums.h"
+#include <limits>
+
+#include "DataFormats/Candidate/interface/LeafCandidate.h"
+#include "DataFormats/L1THGCal/interface/HGCalTriggerCell.h"
+
+DEFINE_EDM_PLUGIN(HGCalVFEProcessorBaseFactory, 
+        HGCalVFEProcessorSums,
+        "HGCalVFEProcessorSums");
+
+
+HGCalVFEProcessorSums::
+HGCalVFEProcessorSums(const edm::ParameterSet& conf) : HGCalVFEProcessorBase(conf),
+  vfeLinearizationImpl_(conf),
+  vfeSummationImpl_(conf),
+  calibration_( conf.getParameterSet("calib_parameters") )
+{ 
+}
+
+void
+HGCalVFEProcessorSums::run(const HGCalDigiCollection& ee,
+                           const HGCalDigiCollection& fh, 
+                           const HGCalDigiCollection& bh, 
+                           l1t::HGCalTriggerCellBxCollection& triggerCellColl, 
+                           const edm::EventSetup& es) 
+{ 
+  calibration_.eventSetup(es);
+
+  std::vector<HGCalDataFrame> dataframes;
+  std::vector<std::pair<DetId, uint32_t >> linearized_dataframes;
+  std::map<HGCalDetId, uint32_t> payload;
+
+  // convert ee and fh hit collections into the same object  
+  if(!ee.empty())
```

Comment:
looks like there's no actual difference in how the three collections are treated (except the out-of-date `HcalDetId` check noted below), so this function can be simplified to take 1 input collection as previously noted

Reply:
Also remnants of the previous implementation. Will be fixed.